docs/reference/release-notes/8.13.0.asciidoc

```diff
@@ -3,6 +3,16 @@
 
 Also see <<breaking-changes-8.13,Breaking changes in 8.13>>.
 
+[[known-issues-8.13.0]]
+[float]
+=== Known issues
+
+* Downsampling persistent state fails to load, preventing upgraded nodes from joining the cluster (issue: {es-issue}106880[#106880])
```

Comment:
`Downsampling persistent state` to `Downsampling persistent tasks`?

Reply:
Nodes upgraded to 8.13.0 fail to start after attempting to load downsample persistent tasks. Upgraded nodes fail to join the cluster and thus failing the upgrade.

Reply:
Done.